### PR TITLE
Add wait after switch to window

### PR DESCRIPTION
**Pull request by YuriGizra**

### CODE REVIEW

**Comment by YuriGizra** [comment]
@amitaibu, ready

File: behat/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -69,7 +69,11 @@ public function iShouldSeeAtLeastOneFlight() {
       $this->getSession()->switchToWindow($windowNames[1]);
     }
 
+    // Add wait after switching window. It prevents page crash
+    $this->getSession()->wait(2000);
     $this->iWaitForCssElement('.ui-datepicker-calendar .highlight');
+    // Add wait to see result, practicaly prevent docker crash.
+    $this->getSession()->wait(2000);
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: this is not needed to run on "production", just for testing, so lets remove it.

**Comment by YuriGizra** [comment]
@amitaibu, done.

**Comment by amitaibu** [comment]
:+1:

**Pull request merged**